### PR TITLE
Added magento traslation

### PR DESCRIPTION
**Pull request by gelanivishal**
## This PR is a:

- [ ] New topic
- [X] Content update
- [X] Content fix or rewrite
- [X] Bug fix or improvement

## Summary

Updated text on release notes.

## Additional information

List all affected URLs 

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.0/release-notes/tech_bull_20x_MasterCard.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ menu_order: 7
 
 *Technical bulletin published on May 19, 2017.*
 
-These instructions apply to anyone running Magento Open Source (formerly Community Edition) or Magento Commerce (formerly Enterprise Edition) version 2.0.x. 
+These instructions apply to anyone running {{site.data.var.ce}} (formerly Community Edition (CE) or {{site.data.var.ce}} (formerly Enterprise Edition (EE) version 2.0.x.
```
Comment (jeff-matthews) [inline guides/v2.0/release-notes/tech_bull_20x_MasterCard.md]:
```suggestion
These instructions apply to anyone running {{site.data.var.ce}} (formerly Community Edition [CE]) or {{site.data.var.ee}} (formerly Enterprise Edition [EE]) version 2.0.x. 
```

File: guides/v2.0/release-notes/tech_bull_21-upgrade.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ menu_order: 3
 ---
 
 
-These instructions apply to anyone upgrading to Magento Open Source (formerly Community Edition) or Magento Commerce (formerly Enterprise Edition)  version 2.1 (including a Release Candidate). 
+These instructions apply to anyone upgrading to {{site.data.var.ce}} (formerly Community Edition (CE) or {{site.data.var.ce}} (formerly Enterprise Edition (EE)  version 2.1 (including a Release Candidate).
```
Comment (jeff-matthews) [inline guides/v2.0/release-notes/tech_bull_21-upgrade.md]:
```suggestion
These instructions apply to anyone upgrading to {{site.data.var.ce}} (formerly Community Edition [CE]) or {{site.data.var.ee}} (formerly Enterprise Edition [EE])  version 2.1 (including a Release Candidate). 
```

File: guides/v2.1/release-notes/tech_bull_21-upgrade.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ menu_order: 3
 ---
 
 
-These instructions apply to anyone upgrading to Magento Open Source (formerly Community Edition) or Magento Commerce (formerly Enterprise Edition)  version 2.1 (including a Release Candidate). 
+These instructions apply to anyone upgrading to {{site.data.var.ce}} (formerly Community Edition (CE) or {{site.data.var.ce}} (formerly Enterprise Edition (EE) version 2.1 (including a Release Candidate).
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/tech_bull_21-upgrade.md]:
```suggestion
These instructions apply to anyone upgrading to {{site.data.var.ce}} (formerly Community Edition [CE]) or {{site.data.var.ee}} (formerly Enterprise Edition [EE]) version 2.1 (including a Release Candidate). 
```

File: guides/v2.1/release-notes/tech_bull_216-imageresize.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ menu_order: 4
 
 *Technical bulletin updated on May 17, 2017.*
 
-These instructions apply to anyone upgrading to Magento Open Source (formerly Community Edition) or Magento Commerce (formerly Enterprise Edition) version 2.1.6 whose installation runs custom themes. This Technical Bulletin provides both a discussion of the issue and a link to our hot fix for 2.1.6. 
+These instructions apply to anyone upgrading to {{site.data.var.ce}} (formerly Community Edition (CE) or {{site.data.var.ce}} (formerly Enterprise Edition (EE) version 2.1.6 whose installation runs custom themes. This Technical Bulletin provides both a discussion of the issue and a link to our hot fix for 2.1.6.
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/tech_bull_216-imageresize.md]:
```suggestion
These instructions apply to anyone upgrading to {{site.data.var.ce}} (formerly Community Edition [CE]) or {{site.data.var.ee}} (formerly Enterprise Edition [EE]) version 2.1.6 whose installation runs custom themes. This Technical Bulletin provides both a discussion of the issue and a link to our hot fix for 2.1.6. 
```

**Comment by gelanivishal** [comment]
@jeff-matthews It's updated.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**